### PR TITLE
1.0

### PR DESCRIPTION
**Pull request by LoganWright**

### CODE REVIEW

File: Tests/FluentTests/Utilities/Atom.swift [hunk 1]
```diff
@@ -17,7 +17,7 @@ struct Atom: Entity {
         groupId = try node.extract("group_id")
     }
 
-    func makeNode() throws -> Node {
+    func makeNode(context: Context = EmptyNode) throws -> Node {
```
Comment (tannernelson) [inline Tests/FluentTests/Utilities/Atom.swift]: Should the default of `= EmptyNode` be added as a protocol extension?

File: Tests/FluentTests/Utilities/User.swift [hunk 2]
```diff
@@ -11,29 +22,18 @@ final class User: Entity {
...
-        }
-    }
-    static func revert(_ database: Fluent.Database) throws {
-        try database.delete(entity)
+    func makeNode(context: Context = EmptyNode) throws -> Node {
+        return try Node(node: [
+            "id": id,
+            "name": name,
+            "email": email
+            ])
```
Comment (tannernelson) [inline Tests/FluentTests/Utilities/User.swift]: Formatting issue.

File: Tests/FluentTests/Utilities/Atom.swift (hunk 1 above)
Comment (LoganWright) [inline Tests/FluentTests/Utilities/Atom.swift]: There's already a default on protocol extension that does this. The default arg is really only useful on implementations that don't throw. For example, you can do `true.makeNode()` w/o throws, but the protocol extension has to throw

Comment (tannernelson) [inline Tests/FluentTests/Utilities/Atom.swift]: Oh interesting. That works, then.

**Pull request merged**